1MI0800224/week2/week2_task1.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+#include <fstream>
+using namespace std;
+
+double ReadFromFile(ifstream& file){
+    if(!file.is_open())

```

Comment:
I'd say it's best not to check the state of a file that's been given to you in this way. This check makes most sense where the file gets opened.